tests/tfm/tfm_regression_test/testcase.yaml

```diff
@@ -24,3 +24,11 @@ tests:
 
   tfm.regression_ipc_lvl2:
     timeout: 200
+
```

Comment:
One of these is a duplication of the tfm.regression_ipc_lvl2 configuration.
If we don't want to rely on the default value in this testcase file I suggest we remove `tfm.regression_ipc_lvl2`.